testsuite/cluster-test/src/experiments/load_test.rs

```diff
@@ -296,18 +299,23 @@ async fn mempool_load_test(
     let mut bytes = 0_u64;
     let mut msg_num = 0_u64;
     let task_start = Instant::now();
+    let mut pending = 0_usize;
     while Instant::now().duration_since(task_start) < duration {
         let msg = libra_mempool::network::MempoolSyncMsg::BroadcastTransactionsRequest {
             request_id: lcs::to_bytes("request_id")?,
             transactions: vec![], // TODO submit actual txns
         };
-        // TODO log stats for bandwidth sent to remote peer to MempoolResult
-        bytes += lcs::to_bytes(&msg)?.len() as u64;
-        msg_num += 1;
-        sender.send_to(vfn, msg)?;
 
-        // await ACK from remote peer
-        let _response = events.select_next_some().await;
+        if pending < NETWORK_CHANNEL_SIZE {
```

Comment:
you probably should move the `let msg = ` inside this if

Reply:
^^

Reply:
I don't think it's resolved?

Reply:
aha, sorry, I thought u meant let msg_num, resolved it now